web/client/components/map/cesium/EditGeoJSONSupport.jsx

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2023, GeoSolutions Sas.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { useRef, useEffect } from 'react';
+import CesiumModifyGeoJSONInteraction from '../../../utils/cesium/ModifyGeoJSONInteraction';
+
+/**
+ * Support for 3D drawing, this component provides only the interactions and callback for a drawing workflow
```

Comment:
Explicitly document the fact that this takes only simple geometries, and propeties and add TODOs to allow the impleemntation of multi-geometries for the future.